BookingsApi/BookingsApi.Domain/Hearing.cs

```diff
@@ -404,6 +404,31 @@ public bool CanAllocate(JusticeUser user, AllocateHearingConfiguration configura
 
             return true;
         }
+
+        public bool IsVhoAllocated()
```

Comment:
as per previous comment, maybe call it `IsJusticeUserAllocated` as allocation is not specific to vhos

Reply:
done